### ISSUE TITLE
Tab, Accordion, and Sidebar - consider supporting arrow key navigation

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7327]
We should discuss whether these components should have a tab based navigation system (current) or if we should move towards an arrow key based navigation.

This was brought up in https://github.com/grommet/grommet/issues/4068
